images/bridge2/cmd/bridge/main.go

```diff
@@ -474,6 +492,22 @@ func (m *Main) Serve(ctx context.Context) {
 		sess := m.loadRequestSession(ctx, w, r)
 		m.serveSessionText(ctx, w, r, sess)
 	})
+	http.HandleFunc("/sessions/{sessID}/events", func(w http.ResponseWriter, r *http.Request) {
+		ctx, cancel := context.WithCancel(ctx)
+		defer cancel()
+		sess := m.loadRequestSession(ctx, w, r)
+		events := sessionEventStream(ctx, sess)
+		m.serveSessionUpdates(ctx, w, r, sess)
+
+		h := w.Header()
+		h.Set("Content-Type", "text/event-stream")
+		h.Set("Cache-Control", "no-store")
+
+		for e := range events {
+			fmt.Fprintf(w, "event: %s\nid: %s\n\n", e.Type, e.ID)
+			w.(http.Flusher).Flush()
```

Comment:
I guess this should be checking for write errors to see if the client disconnected.